public/app.css

```diff
@@ -1,3 +1,5 @@
+@import "../src/colony_table.css";
```

Comment:
This one can slow down the assets loading, because of chained requests, but it's ok in this small project